x-pack/plugins/uptime/server/lib/requests/get_monitor_availability.ts

```diff
@@ -60,6 +60,11 @@ export const getMonitorAvailability: UMElasticsearchQueryFn<
       query: {
         bool: {
           filter: [
+            {
+              exists: {
+                field: 'summary',
+              },
+            },
```

Comment:
When we do this filter in the overview page, we play this dance with using a negate filter instead of a positive filter. It's unclear to me the history of this, and if this is entirely required, but when I worked on changing last successful step to last successful check, I moved this filter to a helper and used it on both overview and last successful check. Do you know why we want with this more complicated filter on overview instead of the simple one here? Want to make sure there's not something we're missing here https://github.com/elastic/kibana/pull/118883/files#diff-b432c7e8ae246a8b600e9ce987ea086765b65ab8290c2252d2947933530cdb5aR18-R43

Reply:
So actually that filter is used in `get_pings` query not the overview monitor list query.  It makes sense because for browser pings, summary document is a ping.

What that filter is actually doing is that we don't want documents which are of browser type and which are not summary documents. it's a double negate filter, if you take a look closely.

Reply:
Right, I understand that it's a double negate filter. What I was wondering, is *why* we used a double negate filter in the first place in that context, and if that reason applies to this context as well. Is the double negate necessary in `get_pings`? What was the history? If so, is it necessary to use the double negate filter here too? Or should we replace that double negate filter with a normal filter in a future enhancement.

Reply:
i think that's a valid point, we can refactor that to only looks for summary documents, i think it's probably a side effect of initial data modal.